### PR TITLE
chore: add tailwind and stitches

### PR DESCRIPTION
**Pull request by csosamorales**

### CODE REVIEW

**Review by zoruka — APPROVED** [review]

**Review by zoruka — APPROVED** [review]

**Comment by jsonsivar** [comment]
Maybe @homoky can eyeball this since he has experience and so we are foundationaly aligned.

**Comment by homoky** [comment]
Pretty good. Very good about having custom props for Button. You should do that with every component that you use anywhere even it is just using in one place. Good example is the `useToast` hook.

You need to have custom Props for components and hooks so when you find out that the css framework, ui kit, hook, alert library, etc is not what you are expecting and you would use anything else, you can and still you wont touch anything else but one file.

👏

**Comment by homoky** [comment]
Feel free to check our codebase, to get some idea how we think about Button component and how we integrate it with story book.

**Comment by homoky** [comment]
Storybook PR - https://github.com/FleekHQ/fleek/pull/261

**Comment by csosamorales** [comment]
@homoky thank you for your review! The button component was just an example from storybook tbh, but yes! We will do all the components with custom props.

thanks for the storybook PR! I'll take a look on that

File: ui/package.json [hunk 1]
```diff
@@ -25,22 +29,37 @@
...
     "eslint-plugin-jest": "^27.1.6",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.31.11",
     "ethers": "^5.7.2",
+    "postcss": "^8.4.21",
     "prettier": "^2.8.0",
     "react-icons": "^4.7.1",
     "react-query": "^3.39.2",
+    "storybook-dark-mode": "^2.0.5",
+    "tailwindcss": "^3.2.4",
```
[the remark this answers is not in the document]
Comment (csosamorales) [reply ui/package.json:63]: tailwind? no, it's just for dev, cause it's generate a file with all the css needed for prod

**Review by zoruka — APPROVED** [review]

**Pull request merged**